src/resmom_win/win/mom_mach.c

```diff
@@ -561,8 +608,10 @@ mem_sum(job *pjob)
 			IsProcessInJob(ptask->ti_hProc, pjob->ji_hJob, &is_process_in_job);
```

Comment:
Please add error logging here

Reply:
I have replied to this comment previously.